README.md

```diff
@@ -1,6 +1,6 @@
 # YOUTUBE-CLONE
 
-> This project is assigned by a plattform (THE ODIN PROJECT) under the (course HTML and CSS).
+> This project followes the guilde line of [THE ODIN PROJECT](https://www.theodinproject.com/courses/html-and-css/lessons/grid) under the (course HTML and CSS) to clone a youtube page [YouTube](https://www.youtube.com/watch?v=Z_sVBF68N9o).
```

Comment:
You should pay attention to incorrect spellings and/or typo errors in your documentation. This is in reference to "*guilde line"* which ought to be *"guideline"*.